File: 217-contains-duplicate/217-contains-duplicate.java
import java.util.HashSet;

class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> numSet = new HashSet<Integer>();
        for (int num : nums) 
        { 
            numSet.add(num);
        }
        return nums.length != numSet.size();
    }
}

File: 217-contains-duplicate/217-contains-duplicate.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Boolean
     */
    function containsDuplicate($nums) {
        $hashSet = [];
        foreach ($nums as $num) {
            if (!array_key_exists($value, $hashSet)) {
                $hashSet[$num] = true;
            }
        }
        return count($hashSet) != count($nums);
    }
}